File: Modul 5/Bahasa C/PRAK503-2310817120001-DinaIzzatiElfadheya.c
#include <stdio.h>

int maksimal(int a, int b) {
    int x = a;
    if(a < b){
        x = b;
}
    return x;
}
int minimal(int a, int b) {
    int y = a;
    if(a > b){
        y = b;
    }
    return y;   
}
int main() {
    int batas = 0;
    int maks = -100000;
    int minim = 100000;
    int bilangan;

    printf("Input\n");
    scanf("%d", &bilangan);
    while (batas < bilangan) {
        int nilai;
        scanf("%d", &nilai);
        maks = maksimal(maks, nilai);
        minim = minimal(minim, nilai);
        batas++;
        }
    printf("\nOutput\n");
    printf("%d %d", maks, minim);
}

File: Modul 5/Bahasa Python/PRAK503-2310817120001-DinaIzzatiElfadheya.py
def maksimal(a, b):
    x = a
    if a < b:
        x = b
    return x

def minimal(a, b):
    y=a 
    if a > b:
        y = b
    return y

print("Input")
batas = 0
maks = -100000
minim = 100000
bilangan = int(input())
nilai = list(map(int,input().split()))

for i in range(batas, bilangan):
    maks = maksimal(maks, nilai[i])
    minim = minimal(minim, nilai[i])
    batas += 1

print("\nOutput")
print(f"{maks} {minim}")



